### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by warrenseine**
You should review commit by commit, not the whole thing together.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -53,6 +53,9 @@ CMAKE_BIN=`which cmake3 || which cmake`
 mkdir -p $BUILDDIR/artifacts/
 mvn ${MAVEN_ARGS} dependency:copy-dependencies -DoutputDirectory=$BUILDDIR/artifacts/
 
+# Create the resource directory where all native libraries will be copied.
+mkdir -p $BUILDDIR/all/
```
Comment (dbouron) [inline build.sh:57]: OPT: surround with braces the variable

File: pom.xml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>JVips</groupId>
-    <artifactId>JVips</artifactId>
+    <groupId>com.criteo</groupId>
```
Comment (dbouron) [inline pom.xml:6]: Good catch! (╯°□°)╯︵◓

File: build.sh (hunk 1 above)
Comment (dbouron) [reply build.sh:57]: Not necessary to change it in this PR as it's a big one

Comment (warrenseine) [reply build.sh]: I've added a follow-up commit: https://github.com/criteo/JVips/pull/44/commits/e052b418153d30cf5c671eb9c91bb8d503cfc5c4

Comment (dbouron) [reply build.sh]: Thank you

**Comment by warrenseine** [comment]
Yes, if GitHub Actions works as expected, Travis becomes redundant.

**Pull request merged**